documentation/modules/proc-iot-logging.adoc

```diff
@@ -0,0 +1,213 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-manual-steps.adoc
+// assembly-installing-kubernetes.adoc
+
+[id='proc-iot-logging-{context}']
+
+= Configuring logging
+
+If the default logging settings are not sufficient, the following
+sections describe ways to configure the logging system.
+
+== Configure the global log levels
+
+The global logging configuration is applied to all
+services which do not have any explicit logging configuration.
+
+By default the global default log level is `info`.
+
+.Prerequisites
+* link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#iot-installing-services-{context}[The IoT services are installed].
+
+.Procedure
+
+. Edit the `IoTConfig` instance named `default`:
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+{cmdcli} edit iotconfig default
+----
+. Configure the logging options, save and exit the editor:
++
+[source,yaml,options="nowrap"]
+----
+apiVersion: iot.enmasse.io/v1alpha1
+kind: IoTConfig
+metadata:
+  namespace: enmasse-infra
+  name: default
+spec:
+  logging:
+    level: info # <1>
+    loggers: # <2>
+      io.enmasse: debug # <3>
+      io.netty: error # <4>
+----
+<1> The default global log level. If omitted, `info` will be used.
+<2> The section for log channel specific entries.
+<3> Lowers the filtering to `debug` level for messages in the channel `io.enmasse`
+<4> Raises the filtering to `error` level for messages in the channel `io.netty`
```

Comment:
So the log filter level or log level ? It feels one is the inverse of the other.